### ISSUE TITLE
[Bug] Video freezes after a while

### ISSUE THREAD

**Issue by Tharn** [omega-gg/MotionBox#35]
After a certain time, the video freezes while the audio continues. Player is still responsive, just the video is a still frame. This does not happen on every video, but it happened repeatably here:  https://www.youtube.com/watch?v=yQg0zNKGUjo

**Comment by 3unjee** [comment]
Yes that's related to the way "input-slave" is currently handled in libVLC. We'll have to wait for it to change on the VLC side.

I have opened a ticket here: https://code.videolan.org/videolan/vlc/-/issues/7264

**Comment by 3unjee** [comment]
This is related -> https://github.com/omega-gg/MotionBox/issues/7

**Comment by oblitum** [comment]
I just installed the app from ArchLinux AUR and every video I've picked to play (searched PBS Space Time and picked the first, then Arvin Ash and again picked the first) from YT results in this problem.

**Comment by 3unjee** [comment]
You mean the last few seconds of the video are cropped ?

**Comment by oblitum** [comment]
No, I mean it behaves just like describe in the issue's initial description, but right at the very few seconds.

**Comment by 3unjee** [comment]
@oblitum can you try the latest version (2.2.0) and let me know if such behavior still happens for you ?
